Makefile

```diff
@@ -117,7 +118,7 @@ dist-binary:
 	go build -o dist/integration/kopia github.com/kopia/kopia
 
 integration-tests: dist-binary
-	KOPIA_EXE=$(CURDIR)/dist/integration/kopia $(GO_TEST) -v -count=1 -timeout 90s github.com/kopia/kopia/tests/end_to_end_test
+	KOPIA_EXE=$(CURDIR)/dist/integration/kopia $(GO_TEST) -count=1 -parallel $(PARALLEL) -timeout 300s github.com/kopia/kopia/tests/end_to_end_test
```

Comment:
Any time reductions or increases due to running the tests in parallel?

Reply:
It's about 50% reduction in wall clock time:

Before:

```
$ time make integration-tests
ok  	github.com/kopia/kopia/tests/end_to_end_test	67.782s
make integration-tests  55.67s user 15.15s system 102% cpu 1:09.38 total
```

After:

```
$ time make integration-tests
ok  	github.com/kopia/kopia/tests/end_to_end_test	35.230s
make integration-tests  93.23s user 19.32s system 305% cpu 36.885 total
```